src/app/kvm/views/LXDSingleDetails/LXDSingleSettings/AuthenticationCard/AuthenticationCard.test.tsx

```diff
@@ -40,59 +34,41 @@ describe("AuthenticationCard", () => {
 
   it("shows a spinner if pod is not PodDetails type", () => {
     state.pod.items[0] = podFactory({ id: 1 });
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/kvm/1/edit", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <AuthenticationCard hostId={pod.id} />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
-    );
-    expect(wrapper.find("Spinner").exists()).toBe(true);
+    renderWithBrowserRouter(<AuthenticationCard hostId={pod.id} />, {
+      route: "/machines",
```

Comment:
Make sure you set the routes correctly, these are different from what was originally specified

```suggestion
      route: "/kvm/1/edit",
```